### DISCUSSION TITLE
press and hold key for certain duration of time

### DISCUSSION THREAD

**Discussion by ZeRenCZ** [sezanzeb/input-remapper#169] [Q&A] [answered]
Hi, I would like to create macros like this:
_**press W key > keep it pressed 10 second > release key**_

I tried everything I could think of, nothing worked

is this possible?

**Comment by cybermaus** [comment]
Maybe `e(EV_KEY,KEY_W,1).w(10000).e(EV_KEY,KEY_W,0)` (I may have the 0 and 1 swapped, they designate keydown and keyup)

**Comment by ZeRenCZ** [comment, marked as answer]
It is exactly what I was looking for, thx :-)
